lib/rhc/helpers.rb

```diff
@@ -132,6 +132,7 @@ def user_agent
       debug certificate_key(value)
     end
 
+    global_option '--ssh STRING', 'Path to a ssh executable and command options to use when performing ssh commands. When providing options, ensure path and options are encased in single or double quotes.', :hide => true
```

Comment:
Given this is a global flag that can also be set permanently in the config file, the `rhc setup` command must be able to receive it (e.g. `rhc setup --ssh /path/to/cmd`) which will save it to the config automatically. Must make sure it works.